project.clj

```diff
@@ -4,7 +4,8 @@
   :license { :name "Eclipse Public License" }
   :source-paths ["src/clojure"]
   :dependencies [[org.clojure/clojure "1.6.0"]
-                 [org.apache.tika/tika-core "1.7"]]
+                 [org.apache.tika/tika-core "1.7"]
+                 [org.apache.tika/tika-parsers "1.7"]]
```

Comment:
if you use tika-parsers, the tika-core will be included via transient dependencies